### ISSUE TITLE
Managed accounts always show "Insufficient Balance" on locks

### ISSUE THREAD

**Issue by cnasc** [unlock-protocol/unlock#4720]
**Describe the bug**
After #4534, where we started checking whether users had a sufficient amount of ERC20 tokens to purchase keys for an ERC20 lock, managed user accounts (which always have a balance of `0` for every token) started to show the "Insufficient Balance" message.

Previously it worked because we did not check for balance on ERC20 locks, we deferred that warning to the wallet.

**To Reproduce**
1. Run local dev
2. Visit the demo page with an ERC20 lock set up and user accounts enabled
3. Log in and go to the checkout
4. See something like the screenshot below

**Expected behavior**
Since user accounts never actually have a balance, we should "fake it" for approved ERC20 locks.

**Screenshots**
<img width="827" alt="image" src="https://user-images.githubusercontent.com/9300702/64703780-1af76e80-d47b-11e9-8fcf-2814c9883a43.png">
